blas/blas64/blas64.go

```diff
@@ -158,6 +158,8 @@ func Swap(x, y Vector) {
 
 // Copy copies the elements of x into the elements of y:
 //  y[i] = x[i] for all i.
+// Note that the behavior of Copy behaves differently than the built-in copy.
```

Comment:
`// Copy requires that the lengths of x and y match and will panic otherwise.` (avoiding behaviour behaving and also the more stark disparities in American v British English).